alignak/daemon.py

```diff
@@ -182,7 +182,9 @@ class Daemon(object):  # pragma: no cover, not for unit tests...
         'server_key':
             StringProp(default='etc/certs/server.key'),
         'ca_cert':
-            StringProp(default='etc/certs/ca.pem'),
+            StringProp(default=''),
+        'server_dh':
```

Comment:
see  https://wiki.openssl.org/index.php/Diffie_Hellman
It's used to better secure the SSL transaction